packages/wonder-blocks-form/src/components/text-field.js

```diff
@@ -198,7 +236,7 @@ const styles = StyleSheet.create({
         },
     },
     error: {
-        background: "rgba(217, 41, 22, 0.06)",
+        background: `linear-gradient(0deg, rgba(217, 41, 22, 0.06), rgba(217, 41, 22, 0.06)), ${Color.white}`,
```

Comment:
Is there any way to compute these RGBA values based on one of our existing colors? Possibly using one of the methods from the `wonder-blocks-color` package?

Reply:
I'll try to see...

Reply:
I did not realize that the WB Color package had such methods. I was able to replicate the same effect using the `mix` and `fade` methods 👍.